hypervisor/lazy.go

```diff
@@ -49,6 +49,13 @@ func statePreparing(ctx *VmContext, ev VmEvent) {
 		ctx.setWindowSize(cmd.ClientTag, cmd.Size)
 	case COMMAND_RUN_POD, COMMAND_REPLACE_POD:
 		glog.Info("got spec, prepare devices")
+		err := ctx.DCtx.(LazyDriverContext).InitVM(ctx)
```

Comment:
create VM in RUNPOD event looks very very wierd. we create&launch vm before RUNPOD event in normal mode. why don't make lazy mode consist with normal mode?